### DISCUSSION TITLE
Quickdraw

### DISCUSSION THREAD

**Discussion by eu-sandra** [community/community#37121] [Other Feature Feedback, Questions, & Ideas]
Como conseguir o Quickdraw?

**Comment by DenverCoder1** [comment]
You can find details about all the achievements here:

https://github.com/Schweinepriester/github-profile-achievements

**Comment by FatChicken277** [comment]
Closing a pull request or issue within 5 minutes of being created

**Comment by SilasRodrigues19** [comment]
@eu-sandra Você precisa abrir uma Pull Request e fechar ela em menos de 5 minutos, fazendo isso é para aparecer de imediato no seu perfil, caso contrário pode ser bug.

**Comment by Khushboqof** [comment]
If you implement this, you will get this badge. appear when you create a new PR and they are lost within 5 minutes of opening. Hope this will be help you.

**Comment by arturyushchenko** [comment]
look here https://github.com/Schweinepriester/github-profile-achievements

**Comment by Khushboqof** [comment]
Closed an issue or a pull request within 5 min of opening)))))))))))))))))
